src/features/fireWeather/FireWeatherPage.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { initOptions } from 'utils/keycloak.json'
+import { selectAuthenticationReducer } from 'app/rootReducer'
+import {
+  createAuthentication,
+  resetAuthentication
+} from './slices/authenticationSlice'
+
+export const FireWeatherPage = () => {
+  const { isAuthenticated } = useSelector(selectAuthenticationReducer)
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    const script = document.createElement('script')
+
+    script.src = `${initOptions.url}/js/keycloak.js`
+    script.async = true
+
+    const keyCloakScript = document.body.appendChild(script)
```

Comment:
You are applying this script every time useEffect is being run, so move this out from React related stuff

Reply:
@Kyubinhan done